components/forms/FileUpload/FileUpload.tsx

```diff
@@ -101,64 +112,82 @@ export default function FileUpload<T extends FieldValues = FieldValues>({
     [name, state.upload, setValue]
   );
 
-  const renderImage = useMemo(() => {
-    return () => {
-      if (!image) return null;
-      const { png } = image;
-      if (!png) return null;
-      const width = 100;
-      const height = 100;
-      const objectFit = "contain";
-      const alt = png.alt ? png.alt : t("forms.file.missingAlt");
-      return (
-        <Image image={{ ...png, alt, width, height }} objectFit={objectFit} />
-      );
-    };
-  }, []);
+  function handleClear() {
+    setIsCleared(true);
+    if (clearName) setValue(clearName, true);
```

Comment:
@scryptmouse I'm hoping you can help me with a typescript error here. In the FileUpload component, I'm trying to set the value of clearName to `true`. clearName's type is `Path<T>`.
```
Argument of type 'boolean' is not assignable to parameter of type 'UnpackNestedValue<PathValue<T, Path<T>>>'.
```